Tools/autotest/arducopter.py

```diff
@@ -9269,6 +9269,83 @@ def EKFSource(self):
         if ex is not None:
             raise ex
 
+    def test_EK3_ext_nav_vel_without_vert(self):
```

Comment:
```suggestion
    def EK3_EXT_NAV_vel_without_vert(self):
```

... at least that's closer to our style for test names....